### PR TITLE
replaced dependency on gopkg.in/logfmt for modules

### PR DESCRIPTION
**Pull request by powerslacker**
`gopkg.in/logfmt` is referenced as both `gopkg.in/logfmt` AND `github.com/go-logfmt/logfmt`.

This creates a mess of issues when using `go mod`. At this point in time it seems like `semver` is being favored by the Go maintainers over gopkg.in style versioning. For now, I've replaced the references to `gopkg.in/logfmt` with `github.com/go-logfmt/logfmt`. I also ran `dep ensure` because I'm unsure how this repo manages dependencies. 

Let me know if there's anything else needed to get this PR moved forward.

### CODE REVIEW

**Comment by jgheewala** [comment]
hi @powerslacker travis build failed. Please pull latest changes from master (https://github.com/signalfx/golib/pull/135/files) and then re-run the travis build.

File: Gopkg.toml [hunk 1]
```diff
@@ -58,10 +58,6 @@
   name = "github.com/go-kit/kit"
   version = "0.7.0"
 
-[[constraint]]
```
Comment (jgheewala) [inline Gopkg.toml]: @powerslacker thanks for noting the inconsistency for logfmt import. However, let us not remove the constraint for this package. If you want to import the package as "github.com/go-logfmt/logfmt", can you please add source option to this constraint. You can look thrift constraint in the same file.

Comment (jgheewala) [reply Gopkg.toml]: Also looking at your code change, we internally found an issue with 

> version = "v0.1.0"

 can you please change the constraint version as below?

> version = "=v0.1.0"

**Comment by mstumpfx** [comment]
Closing as duplicate of #138

**Pull request closed without merging**